backend/src/v5/middleware/permissions/components/teamspaces.js

```diff
@@ -57,4 +58,17 @@ TeamspacePerms.isTeamspaceMember = async (req, res, next) => {
 	}
 };
 
+TeamspacePerms.isModuleEnabled = (moduleName) => async (req, res, next) => {
+	try {
+		const addOns = await getAddOns(req.params.teamspace);
```

Comment:
Have this logic in the teamspaceSettings file and just ask if the module is available here